src/utils/service-requests/playlist-services.js

```diff
@@ -0,0 +1,85 @@
+import axios from "axios";
+
+const getPlaylists = async (dataDispatch) => {
+  try {
+    const token = localStorage.getItem("token");
+    const {
+      data: { playlists },
+    } = await axios.get("/api/user/playlists", {
+      headers: { authorization: token },
+    });
+    dataDispatch({ type: "GET_PLAYLISTS", payload: playlists });
+  } catch (error) {
+    console.log(error);
```

Comment:
remove console.log, Display the error in toast

Reply:
Toast functionality is on my todo list, till then keeping it as it it